crypto/crypto.mbti

```diff
@@ -1,25 +1,31 @@
 package moonbitlang/x/crypto
 
 // Values
-fn arr_u8_to_u32be(Array[Byte], ~i : Int = ..) -> UInt
-
-fn byte_array_to_bytes(Array[Byte]) -> Bytes
+fn arr_u32_to_u8be(Array[UInt]) -> Bytes
 
-fn bytes_to_byte_array(Bytes) -> Array[Byte]
+fn arr_u8_to_u32be(Array[Byte], ~i : Int = ..) -> UInt
 
 fn bytes_to_hex_string(Bytes) -> String
 
+fn bytes_to_iter(Bytes) -> Iter[Byte]
+
 fn chacha12(FixedArray[UInt], UInt, Bytes, ~nonce : UInt = ..) -> Bytes!
 
 fn chacha20(FixedArray[UInt], UInt, Bytes, ~nonce : UInt = ..) -> Bytes!
 
 fn chacha8(FixedArray[UInt], UInt, Bytes, ~nonce : UInt = ..) -> Bytes!
 
-fn md5(Bytes) -> Bytes
+fn md5sum(Bytes) -> Bytes
+
+fn rotate_left(Int, Int) -> Int
```

Comment:
Why is it public?

Reply:
I think rotate_left could be useful in some other scenarios, thus making it public